observations/models.py

```diff
@@ -122,7 +122,9 @@ class Observation(SettingModel):
 
     registrations = models.ManyToManyField(
         Registration,
+        help_text=_("Opnames zijn nooit anoniem en niet te anonimiseren. Let hierop bij het gebruik van de term ‘anoniem’ of ‘geanonimiseerd’ in je documenten voor deelnemers. Voor meer informatie, zie de <a href='https://fetc-gw.wp.hum.uu.nl/wp-content/uploads/sites/336/2021/12/FETC-GW-Richtlijnen-voor-geinformeerde-toestemming-bij-wetenschappelijk-onderzoek-versie-1.1_21dec2021.pdf' target='_blank'>Richtlijnen voor geïnformeerde toestemming, ‘Beeld en geluid’</a>."),
```

Comment:
This `help_text` appears in the "Please specify" field, and as such only when "Other" is selected. Most people won't see it

Reply:
Never mind, this is fixed by Ty in #395 !